Transport.php

```diff
@@ -72,4 +80,43 @@ public static function exists($transport)
     {
         return in_array(strtolower($transport), self::get());
     }
+
+    /**
+     * Register a new wrapper in the collection
+     *
+     * @param  string   $protocol The registered protocol (scheme://).
+     * @param  callable $factory  The factory callable
+     * @return void
+     */
+    public static function registerWrapper($protocol, callable $factory)
+    {
+        static::$_wrappers[$protocol] = $factory;
+
+        return;
+    }
+
+    /**
+     * Check if a wrapper exists in the collection
```

Comment:
Period please :-).

“in the collection”… of what? Either you give more precisions about the collection or you not mention the collection, so: “Check whether a wrapper exists.” is good enough.